ci-operator/config/openshift/installer/openshift-installer-master__okd.yaml

```diff
@@ -187,13 +185,14 @@ tests:
     cluster_profile: gcp
     workflow: openshift-e2e-gcp
 - as: e2e-aws-upgrade
+  skip_if_only_changed: (^docs/)|((^|/)OWNERS(_ALIASES)?$)|((^|/)[A-Z]+\.md$)|(^(data/.*|pkg/.*|images)/(azure|baremetal|gcp|kubevirt|libvirt|openstack|ovirt|vsphere)/)
   steps:
     cluster_profile: aws
-    workflow: openshift-upgrade-aws
+    workflow: openshift-upgrade-aws-latestinstaller
```

Comment:
I don't really know if it's right to use the `-latestinstaller` workflow for OKD. Anyone know some context?

Reply:
I looked at the code and this workflow takes the initial installer from a stable-initial imagestream which I don't think exists for OKD. Reverting this part.